build-caching-maven-samples/webstart-project/pom.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd https://www.gradle.com/gradle-enterprise-maven-project https://www.gradle.com/schema/gradle-enterprise-maven-project.xsd">
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd https://www.gradle.com/develocity-maven-project https://www.gradle.com/schema/gradle-enterprise-maven-project.xsd">
   <modelVersion>4.0.0</modelVersion>
   <artifactId>webstart-project</artifactId>
   <packaging>jar</packaging>
 
   <parent>
     <groupId>com.example</groupId>
-    <artifactId>gradle-enterprise-maven-caching-sample</artifactId>
+    <artifactId>develocity-maven-caching-sample</artifactId>
     <version>1.0-SNAPSHOT</version>
   </parent>
 
```

Comment:
We should remove the namespace on L78 like for the other samples (or at least rename it to Develocity)